src/molecules/Albums.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import { Composition, Box } from 'atomic-layout'
+import styled from 'styled-components'
+import Text from '../atoms/Text'
+
+const StyledImage = styled.img`
+  width: 100%;
+  height: 100%;
+  background-size: cover;
+`
+
+const StyledContainerImg = styled.div`
+  background-color: #000;
+  padding: 5px;
+  width: 100%;
+  height: 100%;
+  overflowe: hidden;
+`
+
+const Albums = ({ data }) => {
+  return (
+    <Composition
+      templateCols="1fr 1fr"
+      templateColsSm="1fr 1fr 1fr"
+      templateColsMd="1fr 1fr 1fr 1fr"
```

Comment:
You can also use `repeat` to have less code:

```diff
- templateColsSm="1fr 1fr 1fr"
+ templateColsSm="repeat(3, 1fr)"
```

It repeats the same value for N amount of times.